components/configuration-mgt/org.wso2.carbon.identity.configuration.mgt.core/src/main/java/org/wso2/carbon/identity/configuration/mgt/core/ConfigurationManagerImpl.java

```diff
@@ -778,6 +778,25 @@ public ResourceFile addFile(String resourceTypeName, String resourceName, String
         return resourceFiles;
     }
 
+    @Override
+    public List<ResourceFile> getFiles(String resourceTypeName, int tenantId) throws ConfigurationManagementException {
+
+        checkFeatureStatus();
+        validateRequest(resourceTypeName);
+        String resourceTypeId = getResourceTypeId(resourceTypeName);
+        List<ResourceFile> resourceFiles = getConfigurationDAO().getFilesByResourceType(resourceTypeId, tenantId);
+        if (CollectionUtils.isEmpty(resourceFiles)) {
+            if (log.isDebugEnabled()) {
+                log.debug("Resource type: " + resourceTypeName + " does not have any files.");
```

Comment:
Is the tenant tracked in some other way (part of the log line)?
If not, would be better to included the tenantId or tenantDomain as part of the log

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/3104/commits/58591c8a79d9819dc4e42f6c1697f7e6061b1682